cmd/integration/integration.go

```diff
@@ -306,37 +306,61 @@ func podRunning(c *client.Client, podNamespace string, podID string) wait.Condit
 }
 
 func runStaticPodTest(c *client.Client, configFilePath string) {
-	manifest := `version: v1beta2
-id: static-pod
+	hostname, _ := os.Hostname()
```

Comment:
Also we should be using the name from the kubelet (in the integration test, it is always "localhost" for files)

----- Original Message -----
> > @@ -306,37 +306,61 @@ func podRunning(c *client.Client, podNamespace
> > string, podID string) wait.Condit
> >  }
> >  
> >  func runStaticPodTest(c *client.Client, configFilePath string) {
> > -	manifest := `version: v1beta2
> > -id: static-pod
> > +	hostname, _ := os.Hostname()
> 
> @smarterclayton just removed os.Hostname from the integration tests because
> it was causing issues running them on a mac.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/5785/files#r26958255